### ISSUE TITLE
Helm chart dependencies do not follow best practices

### ISSUE THREAD

**Issue by Imunhatep** [streamnative/charts#141]
Is your feature request related to a problem? Please describe.
Charts do not follow common guideline: chart templates bring dependencies into the main chart instead of using Helm dependency

Describe the solution you'd like
Move services like prometheus, alert-manager, zookeeper to a chart dependencies, That would simplified helm chart support and usage.

Describe alternatives you've considered
Do not see other options, cause the issue is that Chart structure is inconsistent and does not follow best practices

Additional context
As a chart reference with a lot of dependencies GitLab helm chart could be taken..

**Comment by jiazhai** [comment]
@sijie @wolfstudy Would you please help take a look at this issue?

**Comment by sijie** [comment]
@Imunhatep Agreed! Are you interested in contributing a pull request to improve it?

**Comment by Imunhatep** [comment]
Sorry but atm no. I've switched to a "smaller" pulsar chart for evaluating..

**Comment by sijie** [comment]
@Imunhatep Gotcha. Not sure if you have seen this before or not. But you can disable the components you don't need in the values file (https://github.com/streamnative/charts/blob/master/charts/pulsar/values.yaml#L71) and (https://github.com/streamnative/charts/blob/master/charts/pulsar/values.yaml#L98). So you can get a "smaller" pulsar chart for evaluating. Hope that will help you with the evaluation process.
